src/test/java/com/hocs/test/pages/complaints/BFProgressCase.java

```diff
@@ -124,6 +124,9 @@ private String getStageThatPrecedesTargetStage(String targetStage) {
             case "ESCALATED TO WFM":
                 precedingStage = "TRIAGE";
                 break;
+            case "CCH":
+                precedingStage = "TRIAGE";
+                break;
```

Comment:
I dont think this belongs here. CCH isnt a stage of BF workflow, its the receiving team for a transfer of a BF case to the UKVI complaints workflow (COMP in DECS).
I have a similar note in the scenario structure below, I think we should be explicitly defining the transfer steps, and not confuse these progress methods.